app/Http/Requests/UpdateTaskScheduleRequest.php

```
@@ -0,0 +1,34 @@
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'current_start' => ['nullable', 'date', 'before:current_end'],
+            'current_end' => ['nullable', 'date', 'after:current_start'],
+            'duration_days' => ['nullable', 'integer', 'min:1'],
+            'weight_percent' => ['nullable', 'numeric', 'between:0,100', 'regex:/^\d{1,2}(\.\d{1,2})?$/'],
+            'status' => ['required', Rule::in(TaskStatus::values())]
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Validation\Rule' in method 'rules'. (Clean Code Rules)

(StaticAccess)